.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve Exodus
+title: "[Bug] "
+labels: ["bug", "to confirm"]
+assignees: ""
+---
+
+## Describe the bug
+
+A clear and concise description of what the bug is. Add some log extract so we can see what is happening. Don't forget to add some context.
+
+## To Reproduce
+
+Steps to reproduce the issue.
+
+## Expected behavior
+
+A description of what you expected to happen.
+
+## Version details
+
+- OS: [e.g. Debian 12, OSX]
+- Exodus version: [e.g. `1.28.3`]
+
+## Install method
+
+How did you install Exodus? via Docker? From Pip?
```

Comment:
I'm not sure that makes sense, because almost all the issues raised come from people using the exodus platform, and not a local instance.

Reply:
Yes I have see this thing but I think a template is better than a form, the purpose is only to give a useful example to the person who found a bug. Being confronted with a form is often recalcitrant.

Reply:
I added clarification in the template